_posts/2017-08-24-conduct-report.md

```diff
@@ -0,0 +1,14 @@
+---
+author: Lacey Williams Henschel
+category: General
+date: 2017-08-24 00:00:01
+layout: post
+title: "Code of Conduct Transparency Report"
+---
+
+At DjangoCon US, we take the Code of Conduct very seriously. More important than having a Code of Conduct in the first place is having a plan in place to enforce it. To that end, the DjangoCon US team had several organizers who had discussed in advance how to respond to violations of the Code of Conduct. We are also committed to letting our community know about incidents that occurred at the conference, while also protecting the anonymity of the parties involved.  
+
+- An attendee reported that a speaker's presentation made them feel uncomfortable and excluded. Conference organizers reviewed the presentation and determined that it did not violate the Code of Conduct. 
+- A conference attendee reported that someone not affiliated with the conference was verbally harassing them and someone else, also not affiliated with the conference, in the park close to where our opening reception was held. At the time the report was made, the harasser had already left the area. Conference staff assisted the attendee in contacting local law enforcement and asked them if they needed other help. They declined, and the harasser was not seen at our event again. 
+
+We're grateful to those who reported possible Code of Conduct violations to us, and encourage any attendee who experienced harassment at our conference, witnessed someone else being harassed, or has any questions to email the Code of Conduct response team at [coc@djangocon.us](coc@djangocon.us). 
```

Comment:
It should be [coc@djangocon.us](mailto:coc@djangocon.us) to fix the html checker error. The rest looks good!